# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
# adapt versions to foss/2022a.
# downgrading torch to 1.12.0 from 2.0.1 might be a problem.
diff -ru relion-classranker-b6e751e5cb4205d8e9b36d0ae38c3687b3395acb/requirements.txt relion-classranker-b6e751e5cb4205d8e9b36d0ae38c3687b3395acb_relax_requirements/requirements.txt
--- relion-classranker-b6e751e5cb4205d8e9b36d0ae38c3687b3395acb/requirements.txt	2023-08-22 19:15:52.000000000 +0200
+++ relion-classranker-b6e751e5cb4205d8e9b36d0ae38c3687b3395acb_relax_requirements/requirements.txt	2023-11-16 18:21:03.394135835 +0100
@@ -1,3 +1,3 @@
-torch>=2.0.1
-torchvision>=0.15.2
-numpy>=1.24.4
\ No newline at end of file
+torch>=1.12.0
+torchvision>=0.13.1
+numpy>=1.22.3
